lms/djangoapps/mobile_api/settings.py

```diff
@@ -0,0 +1,7 @@
+'''
+    Settings for Mobile Api
+'''
+
+FB_SETTINGS = {'_FACEBOOK_APP_SECRET': "6c26348ef355f53a531890e980ddc731",
```

Comment:
no - definitely not. 
I've deleted the file in favour of the common.py and aws.py used above.